### PR TITLE
Added self.API.FeatureFlags:GetFeatureFlag("AllowDroppingSwords") to the documentation

### PR DESCRIPTION
**Pull request by marceaupatu123**

### CODE REVIEW

File: docs/included-feature-flags.md [hunk 1]
```diff
@@ -20,4 +20,8 @@ message will be displayed when the admin level is changed.
 
 ## AllowChatCommandExecuting (default: `true`)
 If true, commands can be executed from the chat with
-prefixes used. If false, chat executing will be ignored.
\ No newline at end of file
+prefixes used. If false, chat executing will be ignored.
+
+## AllowDroppingSwords (default: `false`)
+If true, players will be able to drop swords. 
+If false they won't.
```
Comment (TheNexusAvenger) [inline docs/included-feature-flags.md]: Can a comma be added here to be consistent?

File: docs/included-feature-flags.md [hunk 2]
```diff
@@ -20,4 +20,8 @@ message will be displayed when the admin level is changed.
 
 ## AllowChatCommandExecuting (default: `true`)
 If true, commands can be executed from the chat with
-prefixes used. If false, chat executing will be ignored.
\ No newline at end of file
+prefixes used. If false, chat executing will be ignored.
+
+## AllowDroppingSwords (default: `false`)
```
Comment (TheNexusAvenger) [inline docs/included-feature-flags.md]: The wrong default value was noted. It actually defaults to `true` to be backward compatible.
https://github.com/TheNexusAvenger/Nexus-Admin/blob/9ab894f712c18df9eab0a421027dd0eda5b63ca5/src/IncludedCommands/BasicCommands/sword.lua#L26

**Review by TheNexusAvenger — CHANGES_REQUESTED** [review]
I forgot that this page existed. There are 2 changes I would like to see before merging.

File: docs/included-feature-flags.md (hunk 1 above)
Comment (marceaupatu123) [reply docs/included-feature-flags.md]: Done.

File: docs/included-feature-flags.md (hunk 2 above)
Comment (marceaupatu123) [reply docs/included-feature-flags.md]: Done.

**Comment by TheNexusAvenger** [comment]
The updated doc is now up: https://thenexusavenger.github.io/Nexus-Admin/included-feature-flags/#allowdroppingswords-default-true

**Pull request merged**